### PR TITLE
[DOC] Adding examples to get_dupes.py and groupby_topk.py

### PR DESCRIPTION
**Pull request by ethompsy**
# PR Description

Please describe the changes proposed in the pull request:

- Adding examples to functions/get_dupes/py

**This PR contributes to #972 .**

# PR Checklist

Please ensure that you have done the following:

1. [ √ ] PR in from a fork off your branch. Do not PR from `<your_username>`:`dev`, but rather from `<your_username>`:`<feature-branch_name>`.

2. [ √ ] If you're not on the contributors list, add yourself to `AUTHORS.md`.

3. [ √ ] Add a line to `CHANGELOG.md` under the latest version header (i.e. the one that is "on deck") describing the contribution.
    - Do use some discretion here; if there are multiple PRs that are related, keep them in a single line.

# Automatic checks

There will be automatic checks run on the PR. These include:

- Building a preview of the docs on Netlify
- Automatically linting the code
- Making sure the code is documented
- Making sure that all tests are passed
- Making sure that code coverage doesn't go down.

# Relevant Reviewers

Please tag maintainers to review.

- @ericmjl

### CODE REVIEW

File: janitor/functions/get_dupes.py [hunk 1]
```diff
@@ -16,20 +17,55 @@ def get_dupes(
 
     This method does not mutate the original DataFrame.
 
-    Functional usage syntax:
-
     ```python
     df = pd.DataFrame(...)
-    df = get_dupes(df)
+    janitor.get_dupes(df)
     ```
```
Comment (ericmjl) [inline janitor/functions/get_dupes.py]: @ethompsy could we remove this Markdown syntax block, please?

**Review by ericmjl — CHANGES_REQUESTED** [review]
Requesting one change, otherwise all things look great!

File: janitor/functions/get_dupes.py (hunk 1 above)
Comment (ethompsy) [reply janitor/functions/get_dupes.py]: Done!

**Comment by ericmjl** [comment]
@ethompsy I ran into issues being unable to push a modified workflow (updated from `dev`) onto your fork. However, I did figure out how to use configuration to ensure whitespace doesn't become an issue anymore. Could you replace the doctest example using the code block content below, verbatim?

```python

        >>> import pandas as pd
        >>> import janitor
        >>> df = pd.DataFrame({
        ...     "age": [20, 23, 22, 43, 21],
        ...     "id": [1, 4, 6, 2, 5],
        ...     "result": ["pass", "pass", "fail", "pass", "fail"]
        ... })
        >>> df
           age  id result
        0   20   1   pass
        1   23   4   pass
        2   22   6   fail
        3   43   2   pass
        4   21   5   fail
        >>> df.groupby_topk('result', 'age', 3) # Ascending top 3
        ... # doctest: +NORMALIZE_WHITESPACE
                  age  id result
        result
        fail   4   21   5   fail
               2   22   6   fail
        pass   0   20   1   pass
               1   23   4   pass
               3   43   2   pass
        >>> df.groupby_topk('result', 'age', 2, {'ascending':False}) # Descending top 2
        ... # doctest: +NORMALIZE_WHITESPACE
                  age  id result
        result
        fail   2   22   6   fail
               4   21   5   fail
        pass   3   43   2   pass
               1   23   4   pass
```

That should do the trick!

**Comment by ericmjl** [comment]
Actually, the easier/faster thing to do is for me to merge (so we can celebrate :tada:!) and then I'll do a new PR with the fix for the doctest.

**Pull request merged**